content/en/docs/architecture/ci-operator.md

```diff
@@ -197,6 +197,10 @@ By making use of the previously compiled artifacts in the intermediate `pipeline
 cache the Go build. If multiple output `images` exist that rely on a previously built artifact, this caching effect can
 reduce build times dramatically.
 
+Using the images in `base_images` to overwrite the ones in `From` directives from `Dockerfile` is considered best practice
```

Comment:
Is this relevant information? The registry-replacer does it automatically for them or not?

Reply:
https://coreos.slack.com/archives/CBN38N3MW/p1616597325211600

It seems not for the above case.

TBH, i still feel confused after reading the readme so many times.
https://github.com/openshift/ci-tools/tree/master/cmd/registry-replacer

Hit a case yesterday: not related to this PR but the tool registry-replacer
https://coreos.slack.com/archives/CBN38N3MW/p1616537922196600?thread_ts=1616537115.193200&cid=CBN38N3MW
If the tool replaces some image in a ci-op's config, would it mean it has configuration in ocp-build-data repo? I thought yes. But it is no. The replacement could happen when it uses a build image in ocp-build-data.

Reply:
Updated.
It is supposed to be replaced by the replacer.
I will dig to see why it is not.